modules/ROOT/pages/tutorial-updates.adoc

```diff
@@ -103,67 +103,128 @@ virt-install --name=fcos --vcpus=2 --ram=2048 --import \
-ostree diff commit from: 113aa27efe1bbcf6324af7423f64ef7deb0acbf21b928faec84bf66a60a5c933
-ostree diff commit to:   f480038412cba26ab010d2cd5a09ddec736204a6e9faa8370edaa943cf33c932
+[core@localhost ~]$ rpm-ostree db diff
+ostree diff commit from: 6df95bdb (6df95bdb2fe2d36e091d4d18e3844fa84ce4b80ea3bd0947db5d7a286ff41890)
+ostree diff commit to:   a3b08ee5 (a3b08ee51b1d950afd9d0d73f32d5424ad52c7703a6b5830e0dc11c3a682d869)
 Upgraded:
-  bind-libs 32:9.11.13-3.fc31 -> 32:9.11.14-2.fc31
+  btrfs-progs 5.6.1-1.fc32 -> 5.7-1.fc32
+  console-login-helper-messages 0.18.1-1.fc32 -> 0.18.2-1.fc32
+  console-login-helper-messages-issuegen 0.18.1-1.fc32 -> 0.18.2-1.fc32
+  console-login-helper-messages-motdgen 0.18.1-1.fc32 -> 0.18.2-1.fc32
+  console-login-helper-messages-profile 0.18.1-1.fc32 -> 0.18.2-1.fc32
+  crun 0.13-2.fc32 -> 0.14.1-1.fc32
+  crypto-policies 20200610-1.git7f9d474.fc32 -> 20200619-1.git781bbd4.fc32
+  dbus 1:1.12.18-1.fc32 -> 1:1.12.20-1.fc32
+  dbus-common 1:1.12.18-1.fc32 -> 1:1.12.20-1.fc32
+  dbus-libs 1:1.12.18-1.fc32 -> 1:1.12.20-1.fc32
   ...
 ----
 
+== Reverting to the previous version
+
 If the system is not functioning fully for whatever reason we can go back to the previous version:
 
 [source,bash]
 ----
-$ sudo rpm-ostree rollback --reboot
+[core@localhost ~]$ sudo rpm-ostree rollback --reboot
 ----
 
-After logging back in after reboot we can see we are now booted back into the old `31.20200108.3.0` deployment from before the upgrade occurred:
+After logging back in after reboot we can see we are now booted back into the old `32.20200629.3.0` deployment from before the upgrade occurred:
 
 [source,bash]
 ----
-$ rpm-ostree status
+[core@localhost ~]$ rpm-ostree status
 State: idle
-AutomaticUpdates: disabled
 Deployments:
 ● ostree://fedora:fedora/x86_64/coreos/stable
-                   Version: 31.20200108.3.0 (2020-01-09T21:51:07Z)
-                    Commit: 113aa27efe1bbcf6324af7423f64ef7deb0acbf21b928faec84bf66a60a5c933
-              GPGSignature: Valid signature by 7D22D5867F2A4236474BF7B850CB390B3C3359C4
+                   Version: 32.20200629.3.0 (2020-07-10T17:58:03Z)
+                    Commit: 6df95bdb2fe2d36e091d4d18e3844fa84ce4b80ea3bd0947db5d7a286ff41890
+              GPGSignature: Valid signature by 97A1AE57C3A2372CCA3A4ABA6C13026D12C944D0
 
   ostree://fedora:fedora/x86_64/coreos/stable
-                   Version: 31.20200113.3.1 (2020-01-14T00:20:15Z)
-                    Commit: f480038412cba26ab010d2cd5a09ddec736204a6e9faa8370edaa943cf33c932
-              GPGSignature: Valid signature by 7D22D5867F2A4236474BF7B850CB390B3C3359C4
-
+                   Version: 32.20200715.3.0 (2020-07-27T11:36:29Z)
+                    Commit: a3b08ee51b1d950afd9d0d73f32d5424ad52c7703a6b5830e0dc11c3a682d869
+              GPGSignature: Valid signature by 97A1AE57C3A2372CCA3A4ABA6C13026D12C944D0
```

Comment:
Might be useful to inspect `zincati.service` after this step to show that the update service is intelligent to know that the latest update is already present on the system and is not trying to update + reboot into it.

Reply:
Fixed in #152